.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
       NUMPY_VERSION="1.8.2" SCIPY_VERSION="0.13.3"
       SCIKIT_LEARN_VERSION="0.14.1"
       PANDAS_VERSION="0.13.0"
-    - DISTRIB="neurodebian" PYTHON_VERSION="2.7"
+    # - DISTRIB="neurodebian" PYTHON_VERSION="2.7"
```

Comment:
Looking at the sphinx-gallery .travis.yml I figured out how to use Ubuntu 14.04 on Travis. Once you merged this PR, I'll open a new one reenabling neurodebian on 14.04 where everything should work fine.